plugins/inputs/varnish/varnish.go

```diff
@@ -52,10 +56,15 @@ func (s *Varnish) SampleConfig() string {
 }
 
 // Shell out to varnish_stat and return the output
-func varnishRunner(cmdName string) (*bytes.Buffer, error) {
+func varnishRunner(cmdName string, useSudo bool) (*bytes.Buffer, error) {
 	cmdArgs := []string{"-1"}
-
 	cmd := exec.Command(cmdName, cmdArgs...)
+
+	if useSudo {
+		cmdArgs = append([]string{cmdName}, cmdArgs...)
+		cmd = exec.Command("/bin/sudo", cmdArgs...)
```

Comment:
Use the sudo found in path, on my system sudo is `/usr/bin/sudo`, also I think we ought to run sudo with the `-n` flag.